oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/MultiScopesAnalysisListener.java

```diff
@@ -62,7 +61,7 @@
 public class MultiScopesAnalysisListener implements EntryAnalysisListener, ExitAnalysisListener, LocalAnalysisListener {
     private final List<SourceBuilder> entrySourceBuilders = new ArrayList<>(10);
     private final List<SourceBuilder> exitSourceBuilders = new ArrayList<>(10);
-    private final List<DatabaseSlowStatement> slowDatabaseAccesses = new ArrayList<>(10);
+    private final List<SourceBuilder> slowDatabaseAccessesBuilders = new ArrayList<>(10);
```

Comment:
Don't change this. My meaning is, `DatabaseSlowStatement` should have related name fields(used to have IDs only), and do `#prepare` too.

Reply:
@wu-sheng My question is any risk or breakout for my change? I think it' good to handle the sources in the same way.

Reply:
They are not the same thing, we created `DatabaseSlowStatement` intently, SourceBuilder basically relates to the real agent-instralled services.

Reply:
ok, I get your point.